tests/test_slogpy_file.py

```diff
@@ -0,0 +1,21 @@
+"""Import slogpy from slog"""
+from slogpy.slog import Slog as slog # noqa: N813 I001
+import tempfile
+
+"""This is a setup function"""
+def setup_function():
+    """create a file for output"""
```

Comment:
Why is this here, when what it does is overwritten by the initialize in the next function?

Reply:
I don't know why if I don't initialize file_logging = True (slog.initialize(file_logging=True)), it will not create a log file when I run pytest.

Reply:
My bad. I found out why...